pkgs/native_assets_cli/lib/src/api/link_config.dart

```diff
@@ -31,6 +32,11 @@ abstract class LinkConfig implements HookConfig {
   /// `build.dart` script destined for this packages `link.dart`.
   Iterable<Asset> get assets;
 
+  /// The file containing the recorded usages, serialized by
```

Comment:
It doesn't matter how it's produced/serialized - users of this API should only care about how to read / deserialize it.

Consider rephrasing this to something like
```dart
/// The path to the file containing recorded uses after optimizing the application
///
/// The file contents can be parsed using  `package:record_use`.
```